<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubwayWindow</class>
 <widget class="QMainWindow" name="SubwayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>地铁售票与乘车引导系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonStation">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>站点设置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonRoute">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>线路设置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBoxName">
      <property name="text">
       <string>显示站点名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QCheckBox" name="checkBoxDistance">
      <property name="text">
       <string>显示距离</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QCheckBox" name="checkBoxCoordinates">
      <property name="text">
       <string>显示经纬度</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QSlider" name="horizontalSliderZoom">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>623</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>查询</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>起点站</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditStart"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditEnd"/>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QComboBox" name="comboBoxCategory"/>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButtonNavigation">
         <property name="text">
          <string>导航</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QListWidget" name="listWidgetOutput"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>终点站</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="4" colspan="4">
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">background-color: #FFFFFF;</string>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>0</width>
         <height>0</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF;</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="labelCanvas">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #FFFFFF;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pushButtonStation</tabstop>
  <tabstop>pushButtonRoute</tabstop>
  <tabstop>checkBoxName</tabstop>
  <tabstop>checkBoxDistance</tabstop>
  <tabstop>checkBoxCoordinates</tabstop>
  <tabstop>horizontalSliderZoom</tabstop>
  <tabstop>lineEditStart</tabstop>
  <tabstop>lineEditEnd</tabstop>
  <tabstop>comboBoxCategory</tabstop>
  <tabstop>pushButtonNavigation</tabstop>
  <tabstop>listWidgetOutput</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
